app/scripts/modules/amazon/src/serverGroup/configure/wizard/capacity/CapacitySelector.tsx

```diff
@@ -12,12 +12,12 @@ export interface ICapacitySelectorProps {
 }
 
 export class CapacitySelector extends React.Component<ICapacitySelectorProps> {
-  public preferSourceCapacityOptions = [
-    { label: 'fail the stage', value: undefined },
+  private preferSourceCapacityOptions = [
+    { label: 'fail the stage', value: false },
```

Comment:
apparently using `undefined` makes `react-select` show itself as unselected? 🤷‍♂️ I didn't dig deep enough to tell if I was missing something, since this seemed like a reasonable way to get the default to properly show.